docs/api.md

```diff
@@ -212,23 +212,44 @@ the matching. float value between `0` and `1`. `1` is very confident that the ma
 
 **Parameters**
 
 -   `options` **[Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)** Object literal containing parameters for the trip query.
+    -   `options.roundtrip` **\[[Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)]** Return route is a roundtrip. (optional, default `true`)
+    -   `options.source` **\[[String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)]** Return route starts at `any` or `first` coordinate. Can also be `first`. (optional, default `any`)
+    -   `options.destination` **\[[String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)]** Return route ends at `any` or `last` coordinate. Can also be `last`. (optional, default `any`)
     -   `options.steps` **\[[Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)]** Return route steps for each route. (optional, default `false`)
     -   `options.annotations` **\[[Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)]** Return annotations for each route leg. (optional, default `false`)
     -   `options.geometries` **\[[String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)]** Returned route geometry format (influences overview
         and per step). Can also be `geojson`. (optional, default `polyline`)
     -   `options.overview` **\[[String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)]** Add overview geometry either `full`, `simplified` (optional, default `simplified`)
 -   `callback` **[Function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function)** 
 
+**Fixing Start and End Points**
+
+It is possible to explicitely set the start or end coordinate of the trip. When source is set to `first`, the first coordinate is used as start coordinate of the trip in the output. When destination is set to `last`, the last coordinate will be used as destination of the trip in the returned output. If you specify `any`, any of the coordinates can be used as the first or last coordinate in the output.
+
+However, if `source=any&destination=any` the returned round-trip will still start at the first input coordinate by default.
+
+Currently, not all combinations of `roundtrip`, `source` and `destination` are supported.
+Right now, the following combinations are possible:
+
+| roundtrip | source | destination | supported |
+| :-- | :-- | :-- | :-- |
+| true | first | last | **yes** | 
+| true | first | any | **yes** |
+| true | any | last | **yes** |
+| true | any | any | **yes** |
+| false | first | last | **yes** |
+| false | first | any | no |
+| false | any | last | no |
+| false | any | any | no |
```

Comment:
What are our plans wrt. supporting the currently unsupported combinations? Is this getting checked for in code?